projects/authentication/docs/client-access-token.md

```diff
@@ -81,7 +83,7 @@ Authorization: Bearer YOUR_ACCESS_TOKEN
 
 #### More info
 
-publiq currently uses [Auth0](https://auth0.com/) as the implementation of its authentication and authorization service. For more in-depth information about requesting client access tokens, see the [Auth0 documentation for the client\_credentials flow](https://auth0.com/docs/flows#client-credentials-flow).
+publiq currently uses [Keycloak](https://www.keycloak.org/) as the implementation of its authentication and authorization service. For more in-depth information about requesting client access tokens, see the [OAuth 2.0 specification for the client\_credentials flow](https://datatracker.ietf.org/doc/html/rfc6749#section-4.4).
```

Comment:
Is there any reason why we should communicate that we are using keycloak? 
All urls/flows are just oauth compatible, and we do not link to any specific keycloak docs, so I would not give this information away.
This could be valuable info for a hacker with bad intensions.

Reply:
the reason is simple: Auth0 was mentioned and I changed Auth0 to Keycloak. It's a fair point we don't have to mention it. I'll rephrase. 

OTOH it's not a secret (and shouldn't be) that we are using Keycloak. It's obvious in many other ways.

Reply:
Update: we also have this line: https://github.com/cultuurnet/apidocs/pull/408/files#diff-09205b70e866174fcdc81ff1f865f6208b9149eca7d675d3b0bf4589886034b6R15

So if we don't want to talk about keycloak we have to remove that line as well.